src/Service/Resolvers/Sql Query Structures/BaseSqlQueryStructure.cs

```diff
@@ -506,5 +482,67 @@ public void ProcessOdataClause(FilterClause odataClause)
         {
             return filterClause.Expression.Accept<string>(visitor);
         }
+
+        /// <summary>
+        /// Gets the value of the parameter cast as the system type
+        /// </summary>
+        /// <param name="param">Parameter value as a string</param>
+        /// <param name="paramName">Parameter name</param>
```

Comment:
is there a better name for this argument since it may not represent a parameter name? i see one instance where "backingColumnName" is passed in here. Can you also elaborate in the parameter comment that this is only used for error message creation?

Reply:
I've changed `paramName` to `fieldName` and updated the description to reflect that this is used only for constructing the error messages